pkg/connector/connector.go

```diff
@@ -61,6 +61,28 @@ var (
 		Traits:      []v2.ResourceType_Trait{v2.ResourceType_TRAIT_APP},
 		Annotations: v1AnnotationsForResourceType("app", false),
 	}
+	defaultScopes = []string{
+		"okta.users.read",
+		"okta.orgs.read",
+		"okta.groups.read",
+		"okta.roles.read",
+		"okta.profileMappings.read",
+		"okta.policies.read",
+		"okta.myAccount.profile.read",
+		"okta.myAccount.email.read",
+		"okta.apps.read",
+		"okta.appGrants.read",
+		"okta.apiTokens.read",
+		"okta.roles.read",
+	}
+	provisioningScopes = []string{
+		"okta.groups.manage",
+		"okta.roles.manage",
+		"okta.policies.manage",
+		"okta.apps.manage",
+		"okta.appGrants.manage",
+		"okta.apiTokens.manage",
+	}
```

Comment:
Are these all the scopes we need? Are there any extra scopes in this list that we don't need?